bundles/org.openhab.binding.comfoair/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -1587,6 +1606,9 @@
 		<label>Error Current</label>
 		<description>Current errors</description>
 		<category>Text</category>
+		<tags>
+			<tag>Alarm</tag>
+		</tags>
 		<state readOnly="true"/>
 	</channel-type>
```

Comment:
It may lead to questions, if this channel is (pre-)tagged, but not `errors_last`,  `errors_prelast`, and `errors_preprelast`. Obviously, *current errors* are of immediate interest and I would definitely not tag those others with `Alarm` but from an organizational view in the semantic model it may look inconsistent.

Reply:
Again I will probably just delete it.